### PR TITLE
[TASK] Add clarification info for users 

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/js/controllers/app.controller.js [hunk 1]
```diff
@@ -31,6 +31,21 @@ function AppCtrl ($scope, id, net, keychain, txQueue, appManager, rpTracker,
   store.set('announcement', true);
   $scope.showAnnouncement = store.get('announcement');
 
+  // For new terms banner
+  if (store.get('terms') === false) {
```
Comment (vhpoet) [inline src/js/controllers/app.controller.js]: what is this if/else statement doing?

Comment (mrajvanshy) [inline src/js/controllers/app.controller.js]: If the banner has already been dismissed by the user, it shouldn't show up again.

Comment (vhpoet) [inline src/js/controllers/app.controller.js]: this could be simplified to

```
if (store.get('terms') !== false) {
  store.set('terms', true);
}
```

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**